src/common/java/net/minecraftforge/gradle/common/diff/PatchFile.java

```diff
@@ -9,26 +9,32 @@
 
 public class PatchFile {
 
-    public static PatchFile from(String string) {
-        return new PatchFile(() -> new ByteArrayInputStream(string.getBytes(StandardCharsets.UTF_8)), false);
+    public static PatchFile from(String name, String string) {
+        return new PatchFile(name, () -> new ByteArrayInputStream(string.getBytes(StandardCharsets.UTF_8)), false);
```

Comment:
These are all breaking, any particular reason? you need the input name? The target name is where rejects should go.
As in if there are multiple files targeted in a single patch it should spit out multiple rej files.

Reply:
The reason was so that rejects from a single patch file get mapped to a corresponding rej file. I'll change this though.

Reply:
Fixed, turns out the way I was doing it was totally redundant, got a little confused it seems, diff4j is ass.